src/Microsoft.ML.Transforms/Text/TextFeaturizingEstimator.cs

```diff
@@ -279,8 +281,10 @@ public TransformApplierParams(TextFeaturizingEstimator  parent)
             advancedSettings?.Invoke(AdvancedSettings);
 
             _dictionary = null;
-            _wordFeatureExtractor = new NgramExtractorTransform.NgramExtractorArguments();
-            _charFeatureExtractor = new NgramExtractorTransform.NgramExtractorArguments() { NgramLength = 3, AllLengths = false };
+            if (AdvancedSettings.UseWordExtractor)
```

Comment:
> if (AdvancedSettings.UseWordExtractor) [](start = 11, length = 39)

Pr title should have been: Hide the Create methods, change the name of the ImageTransformers to match the conventions used elsewhere, and fix every bug, whitespace and whatnot i see as i work :D